example/src/main/java/com/devcycle/example/java/sdk/app/controller/HelloWorld.java

```diff
@@ -12,10 +14,12 @@
 @Controller
 public class HelloWorld {
 
-    DVCClient dvc;
+    DVCCloudClient dvcCloud;
+    DVCLocalClient dvcLocal;
```

Comment:
should we have separate example apps for Local / Cloud?

Reply:
We think the example app is simple enough that it can be kept together! I did add a custom track event button though